### PR TITLE
[ZH] Update `Main_Page`

### PR DESCRIPTION
**Pull request by cdwcgt**
pp and tournament committees and some other fix
- Tournaments
- Developer

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
is this supposed to be a follow-up to https://github.com/ppy/osu-wiki/pull/6935?

**Comment by cdwcgt** [comment]
> is this supposed to be a follow-up to #6935?

it is only update for commit https://github.com/ppy/osu-wiki/commit/9b7428d718587f9fea31c773261ec4b724b52ac2

File: wiki/Main_Page/zh.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ osu! 的谱面都是充满活力的社区内玩家们的智慧结晶，想与大
 
 社交能把世界各地的 osu! 玩家连结起来。了解这些活动与竞赛，结识新朋友，或帮忙组织一个更好的活动！
 
-[比赛](/wiki/Tournaments) • [聚光灯](/wiki/Beatmap_Spotlights) • [竞赛](/wiki/Contests) • [社区导师计划](/wiki/Community/Community_Mentorship_Program)
+[锦标赛](/wiki/Tournaments) • [聚光灯](/wiki/Beatmap_Spotlights) • [竞赛](/wiki/Contests) • [社区导师计划](/wiki/Community/Community_Mentorship_Program)
```
Comment (cdwcgt) [inline wiki/Main_Page/zh.md:99]:
```suggestion
[比赛](/wiki/Tournaments) • [聚光灯](/wiki/Beatmap_Spotlights) • [竞赛](/wiki/Contests) • [社区导师计划](/wiki/Community/Community_Mentorship_Program)
```
if they have decide use this translate, it need rollback

File: wiki/Main_Page/zh.md [hunk 2]
```diff
@@ -111,12 +111,12 @@ osu! 的谱面都是充满活力的社区内玩家们的智慧结晶，想与大
...
 [核心团队](/wiki/People/The_Team)：[开发团队](/wiki/People/The_Team/Developers) • [全局管理团队 (GMT)](/wiki/People/The_Team/Global_Moderation_Team) • [援助团队](/wiki/People/The_Team/Support_Team) • [审核评估团队 (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) • [谱面审核成员 (BN)](/wiki/People/The_Team/Beatmap_Nominators) • [名人堂](/wiki/People/The_Team/osu!_Alumni) • [社区喜爱计划团队 (LVD)](/wiki/People/The_Team/Project_Loved_Team)
 
-[社区卓越贡献者](/wiki/People/Community_Contributors) • [拥有特殊头衔的玩家](/wiki/People/Users_with_unique_titles)
+[社区贡献者](/wiki/People/Community_Contributors) • [拥有特殊头衔的玩家](/wiki/People/Users_with_unique_titles) • [锦标赛委员会](/wiki/People/Tournament_Committee) • [Performance Points 委员会](/wiki/People/Performance_Points_Committee)
 
 </div>
 <div class="wiki-main-page-panel">
 
-# 开发
+# 开发者
```
Comment (cdwcgt) [inline wiki/Main_Page/zh.md:119]: Need check

File: wiki/Main_Page/zh.md [hunk 3]
```diff
@@ -111,12 +111,12 @@ osu! 的谱面都是充满活力的社区内玩家们的智慧结晶，想与大
 
 [核心团队](/wiki/People/The_Team)：[开发团队](/wiki/People/The_Team/Developers) • [全局管理团队 (GMT)](/wiki/People/The_Team/Global_Moderation_Team) • [援助团队](/wiki/People/The_Team/Support_Team) • [审核评估团队 (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) • [谱面审核成员 (BN)](/wiki/People/The_Team/Beatmap_Nominators) • [名人堂](/wiki/People/The_Team/osu!_Alumni) • [社区喜爱计划团队 (LVD)](/wiki/People/The_Team/Project_Loved_Team)
 
-[社区卓越贡献者](/wiki/People/Community_Contributors) • [拥有特殊头衔的玩家](/wiki/People/Users_with_unique_titles)
+[社区贡献者](/wiki/People/Community_Contributors) • [拥有特殊头衔的玩家](/wiki/People/Users_with_unique_titles) • [锦标赛委员会](/wiki/People/Tournament_Committee) • [Performance Points 委员会](/wiki/People/Performance_Points_Committee)
```
Comment (cdwcgt) [inline wiki/Main_Page/zh.md:114]: the update

File: wiki/Main_Page/zh.md (hunk 2 above)
Comment (kj415j45) [reply wiki/Main_Page/zh.md:119]: revert this change

File: wiki/Main_Page/zh.md (hunk 1 above)
Comment (cdwcgt) [reply wiki/Main_Page/zh.md:99]: image

File: wiki/Main_Page/zh.md (hunk 2 above)
Comment (cdwcgt) [reply wiki/Main_Page/zh.md]: d85e7f5719c98d0e0c2436e394181c371cd2d57

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**